src/rdb.h

```diff
@@ -41,16 +41,15 @@
  *
  * RDB 11 is the last open-source Redis RDB version, used by Valkey 7.x and 8.x.
  *
- * RDB 12+ are non-open-source Redis formats.
+ * RDB 12-79 are reserved for Redis non-compatible RDB formats
```

Comment:
Any particular reason to pick 79? 

With Valkey 9.0 we could have picked 90 as our new RDB version :).

Reply:
This was discussed in the last core-meeting and we decided on 80. Personally I think the 80's (although the bad hair thing) were so cool

Reply:
80+ for Valkey magic wad picked in #1604. It was arbitrary-ish.